.github/workflows/jobs-coral.yaml

```diff
@@ -56,6 +56,9 @@ jobs:
       - name: Setup coral
         uses: ./.github/actions/setup-coral
 
-      - name: Run unittests
+      - name: Get number of CPU cores
+        id: cpu-cores
+        uses: SimenB/github-actions-cpu-cores@v2
+      - name: run tests
         working-directory: ./coral
-        run: pnpm run test-ci
\ No newline at end of file
+        run: pnpm run test-ci --max-workers ${{ steps.cpu-cores.outputs.count }}
```

Comment:
Did you verify that this are the cores for _this_ one workflow or for the whole PR workflows? I would expect this to be the allocated CPUs for all workflows (and that's what I assume), which would mean it makes the tests faster but will make other jobs slower or even fail them with timeouts. (Been there, done that, but with a different dev op tool, so not 100% sure how it is on github)

Reply:
I think it is definitely the cores for the whole github actions flows. I will test out the time that the whole CI flow take to see if it affects other steps.

Reply:
Thanks! As far as I understood the docs, it probably is the same. I've done something similar with a gitlab pipeline once and was REALLY happy how fast the testsuite got... but it slowed down ALL other things and it was reeeeeeaally bad 😅

Reply:
I actually think there is no impact.

Here are all the timings for each step in this PR for the `empty` commit I just pushed:
<img width="981" alt="Screenshot 2023-10-10 at 13 59 37" src="https://github.com/Aiven-Open/klaw/assets/20607294/965fcfbd-9048-49fd-b4a2-3649d4ecbd37">

Here are the timings for all steps on merging the changes from `main` to the PR in https://github.com/Aiven-Open/klaw/pull/1854 :

<img width="925" alt="Screenshot 2023-10-10 at 14 05 36" src="https://github.com/Aiven-Open/klaw/assets/20607294/518dd186-a45f-463d-b9c4-372159608619">

They seem very close (except for FE tests)